cmd/discover.go

```diff
@@ -478,10 +479,40 @@ func (a *OsintScan) InitDiscoverCommand() {
 				a.OutputSignal.AddError(err)
 				return
 			}
+			module, err := cmd.Flags().GetString("module")
+			if err != nil {
+				a.OutputSignal.AddError(err)
+				return
+			}
+			if module != "" {
+				switch strings.ToLower(module) {
+				case "subfinder", "amass", "all":
```

Comment:
Does Go have a dynamic way of adding these entries from something like Fern. TODO: Figure this out before merging in.